committee_admissions/templates/index.html

```diff
@@ -23,9 +23,11 @@
     </noscript>
     <div id="root"></div>
     <script>
-    window.django = {
+    window.__DJANGO__ = {
```

Comment:
This object can be reached by using:
`import djangoData from "src/utils/djangoData";`

Then `djangoData` is `{user: ...}`